.github/workflows/lint.yaml

```diff
@@ -0,0 +1,23 @@
+name: CI
+on: [push, pull_request]
+
+jobs:
+  lint:
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        shell: bash -l {0}
+
+    steps:
+    - uses: actions/checkout@v2
+
+    - uses: conda-incubator/setup-miniconda@v2
```

Comment:
You can use `with` here to specify the file and which environment to activate:
https://github.com/populationgenomics/analysis-runner/pull/5/files#diff-71cabc4177e41ea8f15a89eb65398fa8187739f68a2762706d84885cd8b2ddc3R17

Reply:
The issue with that is that you can't specify the environment name like with `--name`, and you have to use the same name as in the `environment.yaml` file. And I'd love the workflow to be transferrable between projects with copy-and-paste without needing to modify.